examples/deploy_tasks.yaml

```diff
@@ -44,7 +53,27 @@ spec:
             - name: targetCluster
     buildSpec:
         steps:
+        - name: replaceRedisImage
+          image: busybox
+          command: ['sed']
+          args:
+          - "-ri"
+          - "'s/image: k8s.gcr.io\/redis:e2e/image: ${inputs.resources.redisImage.url}@{inputs.resources.redisImage.digest}/' ${inputs.params.pathToFiles}"
+        - name: replaceGuestbookImage
+          image: busybox
+          command: ['sed']
```

Comment:
feedback from @jonjohnsonjr : I'm going to at least TRY using kustomize before I resign myself to `sed` here :D